framework/Cargo.toml

```diff
@@ -40,9 +41,10 @@ twox-hash = ">= 1.2"
 default = []
 performance = []
 sctp = ["rust-sctp"]
+test = ["proptest"]
```

Comment:
we should readme/doc all feature-flags (incl. sctp above haha) w/ example, but yeah.